apollos-church-api/src/config.js

```diff
@@ -7,6 +7,11 @@ ApollosConfig.loadYaml({
   configPath: path.join(__dirname, '..', 'config.yml'),
 });
 
+// defaults
+ApollosConfig.loadJs({
```

Comment:
oh doh 😂 disregard my comment above

Reply:
honestly, this was an experiment, we could throw more in here if they make sense. the goal I think for me is to be completely driven off ENV and not need config 😬

Reply:
totally. Maybe we need to formalize this a bit more? config.yml for values that NEED configuration, .env for values that vary per environment or are secret, and blocks like this for variables w/ sane defaults?